release-notes/6.0/preview/6.0.0-preview.4-runtime-deps.json

```diff
@@ -0,0 +1,1035 @@
+{
+  "dotnetReleaseVersion": "6.0.0-preview.4",
+  "dependencyUsages": {
+    "default": "Used by default or for canonical scenarios",
+    "diagnostics": "Used for diagnostic scenarios such as tracing or debugging",
+    "graphics": "Used for scenarios involving graphics/drawing/fonts",
+    "http.sys": "Used to access the HTTP.sys listener in Windows",
+    "localization": "Used for locale and culture scenarios",
+    "media": "Used for multimedia scenarios",
+    "printing": "Used for printing to printers",
+    "security": "Used for security scenarios",
```

Comment:
How does one figure out whether they app has "security scenarios"? Should these descriptions rather list of the affected types, etc.?

Reply:
This is admittedly a broad way to describe the usage.  There's a balance that needs to be struck here between descriptivity and maintainability.  While I agree that it would be great if we could have a list of affected types, I'm concerned about our ability to keep such a list up-to-date.  I want the maintenance of this file to have low overhead.  Potentially we'll be able to supplement this file with results from static analysis that provides this level of granularity.  But that'd be well down the road.